WAMPClient.spec.js

```
@@ -207,48 +233,26 @@ describe('WAMPClient', () => {
 
 
 				it('should throw an error when no request signature provided', (done) => {
-					const procedure = 'procedureA';
-					const sampleWampServerResponse = Object.assign(someArgument, {
-						procedure,
-						type: WAMPResponseSchema.id,
-						success: false,
-						error: 'err desc',
-						data: {
-							propA: 'valueA',
-						},
-					});
-
-					wampSocket.wampSend(procedure);
+					wampSocket.wampSend(validProcedure);
 					const mockedServerResponse = wampSocket.on.getCalls()[0].args[1];
 					try {
-						mockedServerResponse(sampleWampServerResponse);
+						mockedServerResponse(validWampServerResponse);
 					} catch (err) {
-						expect(err.toString()).equal(`Error: Unable to find resolving function for procedure ${procedure} with signature undefined`);
+						expect(err.toString()).equal(`Error: Unable to find resolving function for procedure ${validProcedure} with signature undefined`);
 						done();
 					}
-
 					done();
 				});
 
-				it('should throw an error when wrong request signature provided', (done) => {
-					const procedure = 'procedureA';
-					const sampleWampServerResponse = Object.assign(someArgument, {
-						procedure,
-						type: WAMPResponseSchema.id,
-						signature: '99999',
-						success: false,
-						error: 'err desc',
-						data: {
-							propA: 'valueA',
-						},
-					});
-
-					wampSocket.wampSend(procedure);
+				it('should throw an error when invalid request signature provided', (done) => {
```

should throw an error when provided an invalid request signature